pkgs/development/libraries/libpam-wrapper/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitLab
+, cmake
+, openpam
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.1.3";
+  pname = "libpam-wrapper";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
+    owner = "cwrap";
+    repo = "pam_wrapper";
+    rev = "ab2d90c6def33888db86755e23f3c12d417e187f";
+    sha256 = "00mqhsashx7njrvxz085d0b88nizhdy7m3x17ip5yhvwsl63km6p";
+  };
+
+  nativeBuildInputs = [ cmake python3 ];
+
+  buildInputs = [ openpam ];
+
+  cmakeFlags = [
+    "-DMAN_INSTALL_DIR=$out/share/man/man1"
```

CMake does not interpolate variables in `-D` flags so it would have to be:

```suggestion
    "-DMAN_INSTALL_DIR=${placeholder "out"}/share/man/man1"
```

But the project uses GNUInstallDirs

https://git.samba.org/?p=pam_wrapper.git;a=blob;f=CMakeLists.txt;h=b453ec361b19ac17dc6f1de6266999d4d4a8cdbe;hb=ab2d90c6def33888db86755e23f3c12d417e187f#l40

so the option would be `CMAKE_INSTALL_MANDIR`, which we already set:

https://github.com/NixOS/nixpkgs/blob/85d6f3bcd9cbcc52c4a307d2ef5116dab4b41641/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L78